build_debian.sh

```diff
@@ -188,7 +188,20 @@ sudo LANG=C DEBIAN_FRONTEND=noninteractive chroot $FILESYSTEM_ROOT apt-get -y in
     usbutils                \
     pciutils                \
     iptables-persistent     \
-    logrotate
+    logrotate               \
+    grub2-common
```

Comment:
can you add grub2-common as into sonic-utilities depends so that we do not need to explicitly specify here.